src/login/logind-dbus.c

```diff
@@ -3440,6 +3497,30 @@ static const sd_bus_vtable manager_vtable[] = {
                                               "b", existing),
                                 method_create_session,
                                 0),
+        SD_BUS_METHOD_WITH_ARGS("CreateSessionWithPIDFd",
```

Comment:
other D-Bus methods we have use `PIDFD` instead of `PIDFd`, so let's keep that naming convention to be consistent